Pages/LanguagePage.cs

```diff
@@ -0,0 +1,113 @@
+﻿using MarsSpecFlow_HannahN.Utils;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MarsSpecFlow_HannahN.Pages
+{
+    internal class LanguagePage 
+    {
+        public void addLanguage(IWebDriver driver)
+        {
+            IWebElement addNew = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[1]/a[1]"));
```

Comment:
Please implement the POM Pattern, please see the below 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html